mod_reforged/hooks/config/strings.nut

```diff
@@ -576,7 +576,7 @@ local vanillaDescriptions = [
 					"Skills build up " + ::MSU.Text.colorNegative("25%") + " less Fatigue.",
 					"[Lash|Skill+lash_skill] and [Hail|Skill+hail_skill] ignore the defense bonus of shields.",
 					"Gain the [From all Sides|Perk+perk_rf_from_all_sides] perk.",
-					"Pound ignores an additional " + ::MSU.Text.colorPositive("+10%") + " of armor on head hits.",
+					"Pound ignores an additional " + ::MSU.Text.colorPositive("+10%") + " of armor on [head hits|Concept.ChanceToHitHead].",
```

Comment:
I think your concept description doesn't always match the use case because of the first sentence and this is one of those cases. I think renaming the concept to Headshots or Critical Hits would be preferable with that first sentence removed. So it would read:

A headshot deals + 50% Hitpoints damage. The default chance to hit the head is 25% but this can be modified by perks or specific attacks. Some enemies lack a head, resulting in all attacks hitting the body.

Reply:
`ChanceToHitHead` is a vanilla concept that is normally only shown when you hover over the headshot chance bar in the stats section of a brother.
It has this text normally: 
> "The base percentage chance to hit a target\'s head for increased damage. The final chance can be modified by the skill used."
But that text is very barebones so I replaced it.

Unless we want multiple concepts around headshots, we need to find one that fits most/all situations.
I agree, the first sentence looks out of place sometimes. But it is pretty useful in one situation; to explain the status bar.

Reply:
Can you use the sentence I wanted to remove only for the status bar?